main/src/ca/uwaterloo/flix/language/phase/Namer.scala

```diff
@@ -82,9 +82,9 @@ object Namer {
        * Namespace.
        */
       case WeededAst.Declaration.Namespace(ns, uses, imports, decls, loc) =>
-        flatMapN(mergeUseEnvs(uses, ns0, uenv0, ienv0, prog0)) {
```

Comment:
Add comments:

// Note: Opening a new namespace clears all current imports and uses.
// Hence we pass empty import and use environments.